ros_buildfarm/templates/snippet/scm_git.xml.em

```diff
@@ -17,7 +17,7 @@
       </hudson.plugins.git.BranchSpec>
     </branches>
     <doGenerateSubmoduleConfigurations>false</doGenerateSubmoduleConfigurations>
-    <submoduleCfg class="list"/>
+    <submoduleCfg class="empty-list"/>
```

Comment:
here is
additionally a final `testDataPublishers` tag with the attribute
"empty-set" I cannot info on what this does but it appears to be a new
feature not an evolution of something we were already doing.